Sources/Testing/ABI/Encoded/ABI.EncodedIssue.swift

```diff
@@ -30,8 +30,17 @@ extension ABI {
     var _severity: Severity
 
     /// Whether or not this issue is known to occur.
+    ///
+    /// This should perhaps be deprecated in favor of `_knownIssueContext`.
     var isKnown: Bool
 
+    /// An ``Issue/KnownIssueContext-swift.struct`` representing the
+    /// ``withKnownIssue(_:isIntermittent:sourceLocation:_:when:matching:)`` call
+    /// that matched this issue, if any.
+    ///
+    /// - Warning: Known issues are not yet part of the JSON schema.
+    var _knownIssueContext: EncodedKnownIssueContext<V>?
```

Comment:
This change does not appear necessary.

Reply:
Even if we do want a `KnownIssueContext` type, this is not the best design for JSON output. The JSON stream's structure is meant to be clean/usable JSON, even if that doesn't exactly match the structure of Swift types in the library.

Reply:
Can you describe how you'd like to represent this info in the JSON?

Reply:
As I wrote in some other comments, it's not even clear to me that we _would_ put it in the JSON.